R/JRC.R

```diff
@@ -0,0 +1,117 @@
+#' R6 Class containing specific attributes and methods for European Commission's
+#' Joint Research Centre data
+#'
+#' @description Information for downloading, cleaning and processing COVID-19
+#' region data from the European Commission's Joint Research Centre
+#'
+#' @source \url{https://github.com/ec-jrc/COVID-19}
+#' @concept dataset
+#' @family national
+#' @export
+#' @examples
+#' \dontrun{
+#' national <- JRC$new(verbose = TRUE, steps = TRUE, get = TRUE)
```

Comment:
it would be nice to go into detail of what is in this class in the examples (i.e show the available regions etc)

Reply:
?